doc/cephadm/operations.rst

```diff
@@ -63,16 +63,6 @@ something like:
 
 This works well for normal operations when logging levels are low.
 
-Disabling logging to stderr
```

Comment:
Can we also remove the "(see above)" from the other remaining disabling logging to stderr section since it's referencing the section being removed


"""
If you choose to log to files, we recommend disabling logging to
stderr ~~(see above)~~ or else everything will be logged twice. Run the
following commands to disable logging to stderr:
"""

Reply:
Done!